### ISSUE TITLE
Add an intentionally failing task which implicates individual files which failed to sync

### ISSUE THREAD

**Issue by alanking** [irods/irods_capability_automated_ingest#296]
When files fail to sync, it will cause the rest of the files in the chunk for that task to fail as well. This could be done simply by allowing the rest of the files in the task to continue syncing.

The failed sync(s) could cause that sync task to fail, which would cause the whole thing to be retried later or require the caller to audit the files in the chunk against objects which successfully synced to the destination collection.

Alternatively, each file which fails to sync could be placed in a new task which intentionally fails but contains information about the file which failed to sync so that it can be retried or at least more easily understood as to what failed.

**Comment by korydraughn** [comment]
I'm assuming the files which follow the failure will be allowed to proceed, correct?

**Comment by alanking** [comment]
Yes, the failure should be caught and then the task should continue processing its list of files.

**Comment by korydraughn** [comment]
Good.

**Comment by alanking** [comment]
Possible duplicate of https://github.com/irods/irods_capability_automated_ingest/issues/184
